lib/syskit/test/profile_assertions.rb

```diff
@@ -308,22 +303,9 @@ def assert_can_instanciate(
                 action_or_profile = subject_syskit_model,
                 exclude: [], together_with: []
             )
-                actions = validate_actions(action_or_profile, exclude: exclude) do |skip|
-                    flunk "could not validate some non-Syskit actions: #{skip}, " \
-                          "probably because of required arguments. Pass the action to " \
-                          "the 'exclude' option of #{__method__}, and add a separate " \
-                          "assertion test with the arguments added explicitly"
-                end
+                actions = assert_actions(action_or_profile, exclude: exclude)
```

Comment:
I think the `assert_actions` and `assert_together_with` names are really not that great...

Maybe `validate_assert_actions_argument` and `validate_assert_together_with_argument` ?